City-of-Helsinki/drupal-hdbt-admin
# UHF-30 Hero toggle functionality

How to test: 
- Check instructions from https://github.com/City-of-Helsinki/drupal-helfi/pull/21 and https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/12
- When the one above is approved, you may approve this one as well
